prisma/migrations/20240531095009_add_activity_log/migration.sql

```diff
@@ -0,0 +1,21 @@
+-- CreateEnum
+CREATE TYPE "ActivityType" AS ENUM ('CREATE_TASK', 'ASSIGN_TASK', 'WORKFLOWSTATE_UPDATE');
+
+-- CreateTable
+CREATE TABLE "ActivityLogs" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "taskId" UUID NOT NULL,
+    "workspaceId" VARCHAR(32) NOT NULL,
+    "activityType" "ActivityType" NOT NULL,
+    "details" JSONB NOT NULL,
+    "deletedAt" TIMESTAMP(3),
```

Comment:
we should add updatedAt as well

Reply:
Adds in 3a1ccc205b027313abe166771613d0e4c9b09125 commit